torchvision/prototype/models/_meta.py

```diff
@@ -1102,6 +1102,28 @@
     "toothbrush",
 ]
 
+# To be replaced with torchvision.datasets.info("coco_kp")
+_COCO_PERSON_CATEGORIES = ["no person", "person"]
```

Comment:
I decided to add this so that the meta have the same minimum attributes across detection. Might remove it on the clean up.